macros/src/interface/generate.rs

```diff
@@ -684,48 +726,50 @@ fn generate_received_request(item_tokens: &mut TokenStream, fizyr_rpc: &syn::Ide
 	let service_id = service.service_id();
 
 	let mut impl_tokens = TokenStream::new();
-
-	if !service.request_updates().is_empty() {
+	if !service.response_updates().is_empty() {
 		generate_send_update_functions(&mut impl_tokens, fizyr_rpc, &quote!(#service_name::RequestUpdate), service.response_updates());
```

Comment:
```suggestion
		generate_send_update_functions(&mut impl_tokens, fizyr_rpc, &quote!(#service_name::ResponseUpdate), service.response_updates());
```
Isnt this correct?

Reply:
Woops, yes. Good catch. The one below is also incorrect I think.

Reply:
Applied. Good catch!